.github/scripts/build.sh

```diff
@@ -50,7 +50,7 @@ git submodule sync && git submodule update --init --recursive
 python -m pip install -r requirements.txt
 python -m pip install mkl-static mkl-include
 export USE_STATIC_MKL=1
-export USE_ONEMKL=1
+export USE_ONEMKL_XPU=1
```

Comment:
Is there any simple API can check whether the oneMKL is compiled and enabled? If have we can add it in the `verify` part of this script

Reply:
These is no such API available. Checking if `libtorch_xpu.so => libmkl_sycl_*.so` is an alternative method.